src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
+    updateCovers(selector);
+  });
+
+  function updateCovers(selector) {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
+    if (highlighterElements.length !== 0) {
```

Comment:
It’s fairly idiomatic to just do `if (highlighterElements.length)` since zero is falsy

Reply:
Also you don’t really need to check the length at all since the `forEach` is just a no-op if it’s empty.

Reply:
Removed